playgrounds/skeleton-svelte/src/routes/components/toast/+page.svelte

```diff
@@ -1,26 +1,26 @@
 <script lang="ts">
-	import { getContext } from 'svelte';
-	import type { ToastContext } from '@skeletonlabs/skeleton-svelte';
+	import { getToastContext } from '@skeletonlabs/skeleton-svelte';
 
-	const toast: ToastContext = getContext('toast');
+	const toastContext = getToastContext();
 
 	function triggerInfo() {
-		toast.create({
-			description: 'This is a basic toast message.',
+		toastContext.toaster.create({
+			// title: 'Info!',
+			description: 'This is an info toast.',
 			type: 'info'
```

Comment:
Nitpick: I like the `type` to come first in our examples. Might help end users a bit if they see that at the top.